lightning/src/ln/features.rs

```diff
@@ -68,7 +68,7 @@ pub type ChannelFeatures = Features<sealed::ChannelContext>;
 impl InitFeatures {
 	/// Create a Features with the features we support
 	#[cfg(not(feature = "fuzztarget"))]
-	pub(crate) fn supported() -> InitFeatures {
+	pub fn supported() -> InitFeatures {
```

Comment:
Looks good, I've been thinking we'd need to do this eventually.

Reply:
I'm not familiar with how the `cfg` attribute works with fuzzing. Given this is now the same as the version below where `#[cfg(feature = "fuzztarget")]` is set, is there a need for the attribute (and thus the conditional definition) still?

Reply:
Oh, right, sorry, yes, also just drop the cfg and duplication, I think.

Reply:
ah, missed the cfg, will remove the dup